tree/dataframe/inc/ROOT/RDFInterface.hxx

```diff
@@ -1744,11 +1744,17 @@ private:
       using NewCol_t = RDFDetail::RCustomColumn<F, CustomColumnType>;
 
       // Declare return type to the interpreter, for future use by jitted actions
-      const auto retTypeName = RDFInternal::TypeID2TypeName(typeid(RetType));
+      auto retTypeName = RDFInternal::TypeID2TypeName(typeid(RetType));
       if (retTypeName.empty()) {
-         const auto msg =
-            "Return type of Define expression was not understood. Type was " + std::string(typeid(RetType).name());
-         throw std::runtime_error(msg);
+         //const auto msg =
+         //   "Return type of Define expression was not understood. Type was " + std::string(typeid(RetType).name());
+         //throw std::runtime_error(msg);
+         const std::string tName(typeid(RetType).name());
+         retTypeName = "unknown_type_" +
+                       tName +
+                       "/* --> Did you forget to inject \"" +
+                       tName +
+                       "\"'s definition in the interpreter?*/";
```

Comment:
so the `Declare` below will try to jit something like
```
"namespace __tdf1 { using x_type = uknown_type_typeidnameoftype; };"
```

How does that `Declare` succeed if `unknown_type_typeidnameoftype` is not a type?